### PR TITLE
Bumper release notes

### PR DESCRIPTION
**Pull request by aludvik**

### CODE REVIEW

File: generator/source/index.md
Comment (annechenette) [inline generator/source/index.md]: Is it bumper or Bumper?

Also, it should be codenamed (not codename).

File: generator/source/release/bumper/notes.md
Comment (dcmiddle) [inline generator/source/release/bumper/notes.md]: :s/:/.

**Review by vaporos — APPROVED** [review]

**Review by dcmiddle — CHANGES_REQUESTED** [review]

**Comment by aludvik** [comment]
@dcmiddle fixed punctuation typo

**Review by dcmiddle — APPROVED** [review]

**Comment by aludvik** [comment]
Rebased on master to pick up Jenkinsfile

**Comment by aludvik** [comment]
Wait until release is live to merge

**Review by annechenette — COMMENTED** [review]
The documentation section appears incomplete - it appears that the only doc changes were to add a Rust SDK and a glossary.  I can provide an additional commit with the formatting changes.

**Comment by aludvik** [comment]
@chenette I think it looks cleaner with the sub-headings.

**Sub-headings**
before

**Sub-lists**
after

**Comment by annechenette** [comment]
I suspect that's because the sub-bullets (dashes) aren't rendering correctly in this view.  I assume that the indenting would be correct on the website, because my online markdown viewers assure me that the sub-bullet indents are correct. Also, it's identical to the format used in the Consensus section.

**Comment by annechenette** [comment]
I stand corrected.  I need more spaces for my indented lists.  I shall push a fix for the Documentation section, then follow with a fix for the Consensus section.

> To nest one list within another, indent each item in the sublist by four spaces.

**Comment by annechenette** [comment]
@aludvik - I wasn't able to get nested (indented) lists working in the "build artifacts". I don't know if this is just an issue with the build environment, but I don't want to risk introducing these format errors in the website.  So I pushed a fixup commit that makes the subheading approach consistent  by changing the first two bullets to subheads. I hope this is an acceptable compromise.

**Review by vaporos — APPROVED** [review]

**Pull request closed without merging**
